packages/utils/src/createBatchedDebounce.test.ts

```diff
@@ -0,0 +1,52 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+import { createBatchedDebounce } from './createBatchedDebounce'
+
+beforeEach(() => {
+  vi.useFakeTimers()
+})
+
+afterEach(() => {
+  vi.useRealTimers()
+})
```

Comment:
thought: Just thinking that what I commonly see in other projects is these life cycle functions are usually placed within the describe block. Could there be a valid reason?